### PR TITLE
OP-1033 Script fix and updates

### PR DESCRIPTION
**Pull request by mizzioisf**
-> corrected a bug on DEMO mode
-> code cleanup
-> code alignment with oh-api scripts - see https://github.com/informatici/openhospital/pull/200

### CODE REVIEW

File: oh.ps1 [hunk 1]
```diff
@@ -522,6 +512,23 @@ function initialize_dir_structure {
 	[System.IO.Directory]::CreateDirectory("$OH_PATH/$BACKUP_DIR") > $null
 }
 
+###################################################################
+function download_file ($download_url,$download_file){
```
Comment (dbmalkovsky) [inline oh.ps1:516]:
```suggestion
function download_file ($download_url,$download_file) {
```
Most times there is a space

File: oh.ps1 [hunk 2]
```diff
@@ -566,23 +567,13 @@ function java_lib_setup {
...
-		Write-Host "Unable to download $download_file from $download_url" -ForegroundColor Red
-		Read-Host; exit 1;
-	}
-	catch {
-		Write-Host "An error occurred. Exiting." -ForegroundColor Red
-		Read-Host; exit 1;
+	
+	# include all jar files under lib\
+	$script:jarlist= Get-ChildItem "$OH_PATH\$OH_DIR\lib" -Filter *.jar |  % { $_.FullName }
+	ForEach( $n in $jarlist ){
```
Comment (dbmalkovsky) [inline oh.ps1:573]:
```suggestion
	ForEach( $n in $jarlist ) {
```
Most times there is a space

File: oh.ps1 (hunk 1 above)
Comment (mizzioisf) [reply oh.ps1]: Fixed !

File: oh.ps1 [hunk 3]
```diff
@@ -136,7 +136,7 @@ $script:DICOM_STORAGE="FileSystemDicomManager" # SqlDicomManager
 $script:DICOM_DIR="data/dicom_storage"
 
 # path and directories
-$script:OH_DIR="."
+$script:OH_DIR="oh"
```
Comment (mwithi) [inline oh.ps1:139]: shouldn't in "gui" be `OH_DIR="."`?

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

**Pull request merged**